### PR TITLE
Fix/Single line dialogue + Refact

### PR DESCRIPTION
**Pull request by ivandrofly**
/cc @niksedk

### CODE REVIEW

File: src/Core/StringExtensions.cs [hunk 1]
```diff
@@ -5,6 +5,33 @@ namespace Nikse.SubtitleEdit.Core
...
+            if (text == null || (!threeLengthTag && !includeFont))
+                return false;
+            var newLineIdx = text.IndexOf(Environment.NewLine, StringComparison.Ordinal);
+            if (newLineIdx < 0 || text.Length < newLineIdx + 5)
+                return false;
+            text = text.Substring(newLineIdx + 2);
+            return StartWithHtmlTag(text, threeLengthTag, includeFont);
+        }
+
+        private static bool StartWithHtmlTag(string text, bool threeLengthTag, bool includeFont)
```
Comment (xylographe) [inline src/Core/StringExtensions.cs]: typo? `StartWithHtmlTag` ==> `StartsWithHtmlTag`?

Comment (ivandrofly) [inline src/Core/StringExtensions.cs]: :+1:

**Comment by ivandrofly** [comment]
@niksedk what should be altered here?!

**Comment by niksedk** [comment]
Don't have too much time atm... so the reviewing will be somewhat slow...

**Pull request merged**